cf-custom-resources/lib/unique-json-values.js

```diff
@@ -92,43 +92,62 @@ const deadlineExpired = function () {
  *  "svc1": ["svc1.com", "example.com"],
  *  "svc2": ["example.com"]
  * }
- *  
- * This function returns a list of unique values found in this map.
+ *
+ * The input event.ResourceProperties.AdditionalAlias is a string (e.g., "foobar.com").
+ *
+ * This function returns a sorted list of unique values found in the map and the additional string.
  * For this example, UniqueValues would be:
- * ["svc1.com", "example.com"]
+ * ["example.com", "foobar.com", "svc1.com"]
  */
 exports.handler = async function (event, context) {
   const responseData = {};
-  const physicalResourceId = event.PhysicalResourceId || event.LogicalResourceId;
+  const physicalResourceId =
+    event.PhysicalResourceId || event.LogicalResourceId;
 
   const handler = async function () {
     switch (event.RequestType) {
       case "Create":
       case "Update":
-        const aliasesForService = JSON.parse(event.ResourceProperties.Aliases || "{}");
-        const filterFor = new Set(event.ResourceProperties.FilterFor.split(","));
+        const aliasesForService = JSON.parse(
+          event.ResourceProperties.Aliases || "{}"
+        );
+        const filterFor = new Set(
+          event.ResourceProperties.FilterFor.split(",")
+        );
         const filteredAliasesForService = Object.fromEntries(
-          Object.entries(aliasesForService).filter(
-            ([key]) => filterFor.has(key)
+          Object.entries(aliasesForService).filter(([key]) =>
+            filterFor.has(key)
           )
         );
         const unique = new Set(Object.values(filteredAliasesForService).flat());
+        if (
+          event.ResourceProperties.AdditionalAlias &&
+          event.ResourceProperties.AdditionalAlias !== ""
+        ) {
```

Comment:
would this ⬇️  be enough?
```suggestion
        if (event.ResourceProperties.AdditionalAlias ) {
```